.circleci/config.yml

```diff
@@ -171,23 +165,18 @@ jobs:
   #                 done
 
   publish-docker-tag:
-    executor: docker-publisher
+    executor: linuxgo
     steps:
       - buildevents/with_job_span:
           steps:
             - checkout
             - setup_remote_docker
+            - run: go install github.com/google/ko@latest
             - run:
-                name: Build Docker image
-                command: docker build -t $IMAGE_NAME:latest .
-            - run:
-                name: Publish Docker Image to Docker Hub
+                name: "Publish multi-arch docker image"
                 command: |
-                  echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
-                  IMAGE_TAG=${CIRCLE_TAG/v/''}
-                  docker tag $IMAGE_NAME:latest $IMAGE_NAME:$IMAGE_TAG
-                  docker push $IMAGE_NAME:latest
-                  docker push $IMAGE_NAME:$IMAGE_TAG
+                  echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdin;
```

Comment:
Why change the env var here from DOCKERHUB_PASS to DOCKER_PASSWORD?

Reply:
I didn't realise I had, but we should be using the username and password stored in the `Honeycomb Secrets for Public Repos` context.

There were duplicates in the project level env vars that can be remvoved.